css-brochure-form/styles.css

```diff
@@ -0,0 +1,75 @@
+body {
+  text-transform: uppercase;
+  font-family: "Brandon Grotesque", sans-serif;
+  font-weight: bold;
+}
+
+h3 {
+  color: orangered;
+  letter-spacing: 0.1rem;
+}
+
+label {
+  font-size: 12px;
+  letter-spacing: 0.05rem;
+  padding-left: 0.5rem;
+}
+
+input {
+  width: 500px;
+  height: 40px;
+  margin-top: 3px;
+  margin-bottom: 15px;
+  border: 2px solid;
+  border-color: lightgray;
+  border-radius: 5px;
+}
+
+input::placeholder {
+  color: rgb(159 157 157);
+  font-size: 14px;
+  padding-left: 0.5rem;
+}
+
+select {
+  width: 500px;
+  height: 40px;
+  margin-top: 3px;
+  margin-bottom: 10px;
+  font-family: "Brandon Grotesque", sans-serif;
+  font-size: 14px;
+  font-weight: 500;
+  border: 2px solid;
+  border-color: lightgray;
+  border-radius: 5px;
+  background-color: rgb(239 239 239);
+}
+
+option {
+  font-family: "Brandon Grotesque", sans-serif;
+  font-size: 14px;
+  font-weight: 500;
+}
+
+button {
+  text-transform: uppercase;
+  font-family: "Brandon Grotesque", sans-serif;
+  font-weight: 500;
+  color: white;
+  background-color: orangered;
+  border-style: none;
+  width: 100px;
+  height: 40px;
+  border-radius: 30px;
+  margin-left: 80%;
+}
+
+button:hover {
+  cursor: pointer;
+}
+
+#flex {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
```

Comment:
Like I mentioned during previous Q&A session, we should avoid styling element with id. Let's change this to use class and we are good to go!